schutzbot/deploy.sh

```diff
@@ -14,7 +14,7 @@ sudo dnf install -y podman \
      qemu-kvm \
      jq
 
-sudo podman pull docker://quay.io/osbuild/postgres:latest
+sudo podman pull docker://quay.io/osbuild/postgres:13-alpine
```

Comment:
Just curious, what's going on here with `13-alpine` instead of `latest`?

Reply:
latest doesn't have multi-arch builds so it was just failing in this PR, and in addition, it is outdated and not built regularly.

Reply:
Ah, thanks!